build/lib/android/snapshot.js

```diff
@@ -34,16 +36,32 @@ async function generateBlob(target) {
 	const V8_LIB_TARGET_DIR = path.resolve(V8_LIB_DIR, target);
 	const MKSNAPSHOT_PATH = path.join(V8_LIB_TARGET_DIR, 'mksnapshot');
 	const BLOB_PATH = path.join(V8_LIB_TARGET_DIR, 'blob.bin');
+	const STARTUP_PATH = path.join(TMP_DIR, 'startup.js');
+	const TI_MAIN_PATH = path.join(TMP_DIR, 'common', 'android', 'ti.main.js');
 	const args = [
-		'--startup_blob=' + BLOB_PATH
+		'--startup_blob=' + BLOB_PATH,
+		STARTUP_PATH,
+		'--print-all-exceptions'
 	];
 
 	// Snapshot already exists, skip...
 	if (await fs.exists(BLOB_PATH)) {
-		console.warn(`Snapshot blob for ${target} already exists, skipping...`);
-		return;
+		const blobTime = fs.statSync(BLOB_PATH).mtime;
```

Comment:
already in an async function, why not `await fs.stat(BLOB_PATH)`? Or better: stat both files in parallel and grab the times:
```javascript
const { blobStat, commonStat} = await Promise.all(fs.stat(BLOB_PATH), fs.stat(TI_MAIN_PATH));
if (commonStat.mtime < blobStat.mtime) {
  return;
}
```

Admittedly, this is small potatoes, but we should try and do things in parallel/async when we can. It's made a huge difference in SDK and app build times.